index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link href="index.css" rel="stylesheet" />
+    <link href="/intro-to-programming-section-5/index.css" rel="stylesheet" />
```

Comment:
You should revert this line back to it's original state. Because index.css is in the same directory as index.html, you don't need to add the "/intro-to-programming-section-5/" into the file href.